util/lambda/index.js

```diff
@@ -0,0 +1,244 @@
+/* eslint-disable no-useless-catch */
+/* eslint-disable no-unused-vars */
+const JSZip = require("jszip");
+const fs = require("fs");
+const AWS = require("aws-sdk");
+const logger = require("../../logger");
+
+const lambda = new AWS.Lambda({
+  apiVersion: "2015-03-31",
+  region: process.env.AWS_REGION || "",
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID || "",
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY || ""
+});
+
+const {
+  createFunction,
+  getFunction,
+  invoke,
+  updateFunctionCode,
+  waitFor
+} = lambda;
+
+const promisify = fn => (...args) =>
+  new Promise((resolve, reject) => {
+    fn(...args, (error, data) => {
+      if (error) {
+        reject(error);
+      } else {
+        resolve(data);
+      }
+    });
+  });
+
+const createZip = async (fileName, code) => {
+  return new Promise((resolve, reject) => {
+    const zip = new JSZip();
+    zip.file(`${fileName}.py`, code);
+
+    zip
+      .generateNodeStream({ type: "nodebuffer", streamFiles: true })
+      .pipe(fs.createWriteStream(`${fileName}.zip`))
```

Comment:
Is there a need to write to the file? How about returning the byte stream from here?

Reply:
lambda is unable to read bytestream sent directly. hence reading file and then sent